content/learn/migration-guides/_index.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Migration Guides"
+sort_by = "weight"
+template = "migration-guide-section.html"
+page_template = "migration-guide-section.html"
+redirect_to = "learn/migration-guides/index"
```

Comment:
```suggestion
redirect_to = "learn/migration-guides/introduction"
```
This is needed to correctly redirect https://bevyengine.org/learn/migration-guides to https://bevyengine.org/learn/migration-guides/introduction (which functions as the index, an actual /index doesn't exist).

(see advice on this page: https://www.getzola.org/themes/book/)